packages/cbioportal-frontend-commons/src/components/StatusHelpers.tsx

```diff
@@ -0,0 +1,34 @@
+import DefaultTooltip from './defaultTooltip/DefaultTooltip';
```

Comment:
Do we need this file if oncokb-frontend-commons already have? @onursumer

Reply:
Ideally we should only have one copy of this. As long as we don't introduce a circular dependency I think it's okay to import it from `oncokb-frontend-commons`.